src/components/calcite-accordion/calcite-accordion.e2e.ts

```diff
@@ -130,6 +130,41 @@ describe("calcite-accordion", () => {
     expect(await item3Content.isVisible()).toBe(true);
   });
 
+  it("clicking on the nested child accordion-item does not toggle the parent in single selection mode", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
```

Comment:
Can you use the [`html` helper](https://github.com/Esri/calcite-components/blob/master/src/tests/utils.ts#L119-L137) to have Prettier handle formatting of embedded JS?

```ts
await page.setContent(html`
// ...
```